chain/vm/invoker.go

```diff
@@ -58,6 +70,18 @@ func NewInvoker() *Invoker {
 	inv.Register(builtin0.VerifiedRegistryActorCodeID, verifreg0.Actor{}, verifreg0.State{})
 	inv.Register(builtin0.AccountActorCodeID, account0.Actor{}, account0.State{})
 
+	inv.Register(builtin1.SystemActorCodeID, system1.Actor{}, abi.EmptyValue{})
+	inv.Register(builtin1.InitActorCodeID, init1.Actor{}, init1.State{})
+	inv.Register(builtin1.RewardActorCodeID, reward1.Actor{}, reward1.State{})
+	inv.Register(builtin1.CronActorCodeID, cron1.Actor{}, cron1.State{})
+	inv.Register(builtin1.StoragePowerActorCodeID, power1.Actor{}, power1.State{})
+	inv.Register(builtin1.StorageMarketActorCodeID, market1.Actor{}, market1.State{})
+	inv.Register(builtin1.StorageMinerActorCodeID, miner1.Actor{}, miner1.State{})
+	inv.Register(builtin1.MultisigActorCodeID, msig1.Actor{}, msig1.State{})
+	inv.Register(builtin1.PaymentChannelActorCodeID, paych1.Actor{}, paych1.State{})
+	inv.Register(builtin1.VerifiedRegistryActorCodeID, verifreg1.Actor{}, verifreg1.State{})
+	inv.Register(builtin1.AccountActorCodeID, account1.Actor{}, account1.State{})
```

Comment:
We really should implement something to bound epochs at which given code CIDs are valid